### PR TITLE
ServerAudit: use IsAdminUser property

### PR DESCRIPTION
**Pull request by eszkadev**
and set old is_admin inside UserExtraInfo as deprecated

TODO: SDK update so links will point to the content

### CODE REVIEW

**Review by caolanm — APPROVED** [review]
looks reasonable to me

File: browser/src/control/Control.ServerAuditDialog.ts [hunk 1]
```diff
@@ -25,11 +25,18 @@ class ServerAuditDialog {
...
-					'SDK: userextrainfo',
-					'https://sdk.collaboraonline.com/docs/advanced_integration.html?highlight=userprivateinfo#userextrainfo',
+					_('The IsAdminUser property is not set'),
+					'SDK: IsAdminUser',
+					'https://sdk.collaboraonline.com/docs/advanced_integration.html?highlight=IsAdminUser#isadminuser',
 				],
-				ok: [_('The is_admin user property is set'), '', ''],
+				deprecated: [
+					_(
+						'Used deprecated id_admin field, please use IsAdminUser property instead',
```
[the remark this answers is not in the document]
Comment (timar) [reply browser/src/control/Control.ServerAuditDialog.ts:34]: Also the suggestion: "please use IsAdminUser property instead" – what can a sysadmin do? "Poke your software vendor to fix this!" ;)

[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/control/Control.ServerAuditDialog.ts]: added "integration should use xxx field" so it is more clear that it is not admin who has to do something :)

**Comment by eszkadev** [comment]
it also fixes random failures in cpp tests caused by commit https://github.com/CollaboraOnline/online/commit/1a0d289e13a5395079209ca38999814eef05cf57
Added API to send server audit information

**Pull request merged**